pyvenv.el

```diff
@@ -145,6 +154,34 @@ This is usually the base name of `pyvenv-virtual-env'.")
 (defvar pyvenv-old-eshell-path nil
   "The old eshell path before the last activate.")
 
+
+(defun pyvenv-create ()
+  "Create virtualenv."
+  (interactive)
+  (let ((create-venv-command)
+        (python-executable)
+        (venv-name)
+        (venv-dir))
+    (run-hooks 'pyvenv-pre-create-hooks)
```

Comment:
Would it make sense to run this hook just before the shell command, not before reading input?